data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java

```diff
@@ -80,7 +117,16 @@ public static List<DataFile> listPartition(
       Configuration conf,
       MetricsConfig metricsConfig,
       NameMapping mapping) {
-    return listPartition(partition, uri, format, spec, conf, metricsConfig, mapping, 1);
+    return listPartition(
+        partition,
+        uri,
+        format,
+        spec,
+        conf,
+        metricsConfig,
+        mapping,
+        1,
+        false /* skipOnErrorOrNot */);
```

Comment:
Comments should be formatted using separate words.

Reply:
done.